Bio/SeqIO/InsdcIO.py

```diff
@@ -604,6 +604,18 @@ def _get_data_division(self, record):
         assert len(division) == 3
         return division
 
+    def _get_topology(self, record):
+        """Set the topology to 'circular', 'linear' if defined"""
+        max_topology_len = len('circular')
+
+        # return an empty placeholder string if not given
+        if 'topology' not in record.annotations:
+            return ' ' * max_topology_len
+
+        template = '%%-%ds' % max_topology_len
```

Comment:
Would the string justification method be easier? i.e. ``.ljust(max_topology_len)``